### DISCUSSION TITLE
Cursor warp option on lazy.screen_next()

### DISCUSSION THREAD

**Discussion by justinesmithies** [qtile/qtile#3919] [Ideas]
Is it possible to add a cursor warp option to the `lazy.next_screen(),` please So when moving focus to the next screen the cursor warps too. I don't want to enable cursor warp in the config as that makes the cursor warp to dunst notifications too which I don't want.

What I'd like is `lazy.next_screen(warp = True),`

**Comment by elParaguayo** [comment]
As far as I can tell, the cursor will warp to the screen if there are no windows on that screen. So, are you asking to have an option where, when there are windows on that screen, the cursor warps to the screen but not the current window?

**Reply by justinesmithies** [reply]
ATM using Wayland if I set cursor warp in the config the cursor jumps into dunst notifications which is an unwanted behaviour. So I thought having lazy.screen_next(warp=True) could just move focus to the next screen and move the cursor to the active window in that screen or if none the center of the screen.

**Reply by elParaguayo** [reply]
The dunst problem sounds like it may need to be fixed separately. I'll have more of a think about the other bit.

**Comment by justinesmithies** [comment]
It's probably a Wayland issue because if you read the dunst config docs they say:

```
follow (values: [none/mouse/keyboard] default: none)

    Defines where the notifications should be placed in a multi-monitor setup. All values except none override the monitor setting.

    On Wayland there is no difference between mouse and keyboard focus. When either of the is used, the compositor will choose an output. This will generally be the output last interacted with.

    none

        The notifications will be placed on the monitor specified by the monitor setting.
    mouse

        The notifications will be placed on the monitor that the mouse is currently in.
    keyboard

        The notifications will be placed on the monitor that contains the window with keyboard focus.
```

**Reply by elParaguayo** [reply]
Regardless of where the notification is displayed, presumably you don't want the cursor to warp to it.

**Reply by justinesmithies** [reply]
So I was bearing off of the reason I asked for the change.
On Wayland rofi or fuzzel open on the screen where the cursor is. lazy.next_screen() doesn't move the cursor to the next window although focus is moved. So rofi or fuzzel open on the wrong window. Cursor warp in config was an option but it warps to every notification too. See my issue hence request.

**Comment by justinesmithies** [comment]
Exactly
